packages/docs/docs/home.md

```diff
@@ -20,20 +20,42 @@ Medplum is a **headless EHR**. Using [Medplum products](./products) you can buil
 
 ![Medplum system overview](/img/medplum-overview.svg)
 
-## Getting Started
+## Get Started
 
-- Get started right away, you can [register here](https://app.medplum.com/register). If needed, Medplum also supports [self-hosting](./docs/self-hosting), get the [source code](https://github.com/medplum/medplum) on Github.
-- The [Basic Concepts](./docs/fhir-basics) page provides a good primer on Medplum and contains important information about the FHIR, the healthcare data standard on which Medplum is built.
-- If you're ready to start coding, use our [tutorials](./docs/tutorials) to get up and running
-- If you want to test out the full Medplum stack locally, see how how you can [run the stack with just two commands!](./docs/self-hosting/running-full-medplum-stack-in-docker)
-- Our detailed documentations goes step-by-step through specific use cases. Don't see a how-to guide you need? Reach out on our [Discord](https://discord.gg/medplum) or send us an email at [support@medplum.com](mailto:support@medplum.com)
+1. [Register an account](https://www.medplum.com/docs/tutorials/register) to get a project provisioned. 
+    1. [Optional]: [Import sample data](https://www.medplum.com/docs/tutorials/importing-sample-data). 
+2. View and use our [provider app](https://www.medplum.com/docs/provider) by [logging in](https://provider.medplum.com/) to use our out-of-the-box provider experience. 
```

Comment:
We us the term "provider app" to contrast with patient apps, etc., but I wonder if the the term "out of the box provider experience" makes sense to newcomers. Suggesting "EHR experience", or "clinician Experience" 

```suggestion
2. Use our [provider app](https://www.medplum.com/docs/provider) by logging in to our [out-of-the-box EHR experience](https://provider.medplum.com/) . 
```